src/app/jscaip/PlayerMap.ts

```diff
@@ -3,50 +3,46 @@ import { MGPOptional } from '../utils/MGPOptional';
 import { Player } from './Player';
 import { PlayerNumberTable } from './PlayerNumberTable';
 
-export class PlayerMap<T extends NonNullable<unknown>> extends MGPMap<Player, T> {
+export class PlayerMap<T extends NonNullable<unknown>> {
 
-    public static of<T extends NonNullable<unknown>>(playerZeroValue: T, playerOneValue: T): PlayerMap<T> {
+    public static ofValues<T extends NonNullable<unknown>>(playerZeroValue: T, playerOneValue: T): PlayerMap<T> {
```

Comment:
why not of ?

Reply:
Car celui là on l'utilise peu, et si on l'appelle `of`, on peut pas faire `PlayerNumberMap.of`, qu'on utilise moulte moulte pluss